spec/fortnox/api/repositories/invoice_spec.rb

```diff
@@ -90,12 +107,6 @@ def save_invoice(country:, vcr_cassette: country)
         it { is_expected.to eq('GB') }
       end
 
-      describe 'KR' do
```

Comment:
Apparently, this was not a valid call anymore. Fortnox complained about it...

Reply:
I checked the VCR cassette and apparently they are still using the old country name: `Korea, Republic of`. When we upgraded the dependencies, we upgraded the `countries` gem, that hopefully has the correct translation of `KR` to `South Korea`, so I guess Fortnox is just slow on such updates...

Reply:
I'll remove the cassette. Since we don't use South Korea we can just ignore this. If anyone needs South Korea, let's fix that then...

Reply:
👍